### PR TITLE
TimescaleDB 2.4 Release notes

### PR DESCRIPTION
**Pull request by NunoFilipeSantos**
# Description
TimescaleDB 2.4 Release notes

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes #[insert issue link, if any]

### CODE REVIEW

File: timescaledb/overview/release-notes/index.md [hunk 1]
```diff
@@ -34,23 +31,34 @@ while also empowering users with more control and flexibility to customize
...
+provide early feedback and influence the direction of TimescaleDB’s development.
+ We’ll travel faster with your input! 
+	
+Please create your feedback as a GitHub issue (using the experimental-schema label), describe what you found, and tell us the steps or share the code snip to recreate it.
 
 Several bugs fixed, see the release notes for more details.
 
 **PostgreSQL 11 deprecation announcement**
 
+PostgreSQL 11 deprecation announcement
```
Comment (mfreed) [inline timescaledb/overview/release-notes/index.md:55]: I think line 55 ("PostgreSQL 11 deprecation announcement") should be removed -- repeats lines 53 ("**PostgreSQL 11 deprecation announcement**")

File: timescaledb/overview/release-notes/index.md [hunk 2]
```diff
@@ -34,23 +31,34 @@ while also empowering users with more control and flexibility to customize
...
 Timescale is working hard on our next exciting features. 
-To make that possible, we require functionality 
-that is available in Postgres 12 and above. 
-For this reason, TimescaleDB 2.3.1 is the final version that supports PostgreSQL 11.  
-TimescaleDB 2.4 will require PostgreSQL 12 or 13.
+To make that possible, we require functionality that is available in Postgres 12 and above. 
+For this reason, from TimescaleDB 2.4 onwards PostgreSQL 12 or 13 are required.
```
Comment (mfreed) [inline timescaledb/overview/release-notes/index.md:58]: For this reason, we are removing support for PostgreSQL 11 starting in the TimescaleDB 2.4 release.  For
TimescaleDB 2.4 and onwards, PostgreSQL 12 or 13 are required.

File: timescaledb/overview/release-notes/index.md [hunk 3]
```diff
@@ -34,23 +31,34 @@ while also empowering users with more control and flexibility to customize
...
+hypertable setup. This includes the ability to add a data node and move
+chunks to the new data node for cluster rebalancing.
+* The `time_bucket_ng` function, a newer version of `time_bucket`. This function
+supports years, months, days, hours, minutes, and seconds.
+ 
+We’re committed to developing these experiments, giving the community a chance to 
+provide early feedback and influence the direction of TimescaleDB’s development.
+ We’ll travel faster with your input! 
+	
+Please create your feedback as a GitHub issue (using the experimental-schema label), describe what you found, and tell us the steps or share the code snip to recreate it.
```
Comment (mfreed) [inline timescaledb/overview/release-notes/index.md:49]: snip -> snippet

**Review by mfreed — APPROVED** [review]

File: timescaledb/overview/release-notes/index.md [hunk 4]
```diff
@@ -15,11 +15,8 @@ we welcome you to visit our Github repo or join our [Slack community](https://sl
 
 For our next release, we plan to add:
 
-- Elasticity: Move data when expanding or shrinking a multi-node cluster
-- Experimental schema: A way to get experimental functionalities early to our users
-- Various bug fixes.
-
-The current generally available (GA) version is 2.3.1.
+- Continuous aggregates for distributed hypertables
```
Comment (gayyappan) [inline timescaledb/overview/release-notes/index.md:18]: Are we committing to adding both features for the next release?

File: timescaledb/overview/release-notes/index.md (hunk 2 above)
Comment (gayyappan) [inline timescaledb/overview/release-notes/index.md:58]: __
```suggestion
For this reason, PostgreSQL 12 or 13 is required for TimescaleDB 2.4 and onwards.
```

File: timescaledb/overview/release-notes/index.md (hunk 3 above)
Comment (NunoFilipeSantos) [reply timescaledb/overview/release-notes/index.md:49]: Corrected here and on the blogpost.
```suggestion
Please create your feedback as a GitHub issue (using the experimental-schema label), describe what you found, and tell us the steps or share the code snippet to recreate it.
```

File: timescaledb/overview/release-notes/index.md (hunk 1 above)
Comment (NunoFilipeSantos) [reply timescaledb/overview/release-notes/index.md:55]:
```suggestion
```

File: timescaledb/overview/release-notes/index.md (hunk 2 above)
Comment (NunoFilipeSantos) [reply timescaledb/overview/release-notes/index.md:57]:
```suggestion
For this reason, we are removing support for PostgreSQL 11 starting in the TimescaleDB 2.4 release. For TimescaleDB 2.4 and onwards, PostgreSQL 12 or 13 are required.
```

File: timescaledb/overview/release-notes/index.md [hunk 5]
```diff
@@ -70,6 +77,29 @@ accidentally triggering the load of a previous DB version.**
 
 ## Unreleased
 
+## 2.4.0 (2021-07-30)
+
+**Experimental Features**
```
Comment (Loquacity) [inline timescaledb/overview/release-notes/index.md:82]:
```suggestion
**Experimental features**
```

File: timescaledb/overview/release-notes/index.md [hunk 6]
```diff
@@ -70,6 +77,29 @@ accidentally triggering the load of a previous DB version.**
...
+* #3293 Add timescaledb_experimental schema
+* #3302 Add block_new_chunks and allow_new_chunks API to experimental
+schema. Add chunk based refresh_continuous_aggregate.
+* #3211 Introduce experimental time_bucket_ng function
+* #3366 Allow use of experimental time_bucket_ng function in continuous aggregates
+* #3408 Support for seconds, minutes and hours in time_bucket_ng
+
+**Bugfixes**
```
Comment (Loquacity) [inline timescaledb/overview/release-notes/index.md:90]:
```suggestion
**Bug fixes**
```

File: timescaledb/overview/release-notes/index.md [hunk 7]
```diff
@@ -34,23 +31,33 @@ while also empowering users with more control and flexibility to customize
...
+hypertable setup. This includes the ability to add a data node and move
+chunks to the new data node for cluster rebalancing.
+* The `time_bucket_ng` function, a newer version of `time_bucket`. This function
+supports years, months, days, hours, minutes, and seconds.
+ 
+We’re committed to developing these experiments, giving the community a chance to 
+provide early feedback and influence the direction of TimescaleDB’s development.
+ We’ll travel faster with your input! 
+	
+Please create your feedback as a GitHub issue (using the experimental-schema label), describe what you found, and tell us the steps or share the code snippet to recreate it.
```
Comment (Loquacity) [inline timescaledb/overview/release-notes/index.md:49]:
```suggestion
Please create your feedback as a GitHub issue, and add the `experimental-schema` label.
Describe what you found, and tell us the steps or share the code snippet to recreate it.
```

**Review by Loquacity — APPROVED** [review]
A couple of non-blocking changes.

File: timescaledb/overview/release-notes/index.md (hunk 4 above)
Comment (NunoFilipeSantos) [reply timescaledb/overview/release-notes/index.md:18]: These are current priorities. :)

**Pull request merged**